deployment/chainloop/values.yaml

```diff
@@ -485,13 +485,26 @@ controlplane:
   ##       ...
   ##       -----END RSA PRIVATE KEY-----
   ## @param controlplane.keylessSigning.fileCA.keyPass The secret key pass
+  ## @param controlplane.keylessSigning.ejbcaCA.serverUrl The url of the EJBCA service (https://host/ejbca)
+  ## @param controlplane.keylessSigning.ejbcaCA.key PEM-encoded the private key for EJBCA cert authentication
+  ## @param controlplane.keylessSigning.ejbcaCA.cert PEM-encoded certificate for EJBCA cert authentication
+  ## @param controlplane.keylessSigning.ejbcaCA.certProfileName Name of the certificate profile to use in EJBCA
+  ## @param controlplane.keylessSigning.ejbcaCA.endEntityProfileName Name of the Entity Profile to use in EJBCA
+  ## @param controlplane.keylessSigning.ejbcaCA.caName Name of the CA issuer to use in EJBCA
   keylessSigning:
```

Comment:
You need to execute the documentation generator to update the readme too https://github.com/bitnami/readme-generator-for-helm

This is how I do it

```
./bin/index.js --values ~/work/chainloop/chainloop/deployment/chainloop/values.yaml --readme ~/work/chainloop/chainloop/deployment/chainloop/README.md
```

Reply:
Done in https://github.com/chainloop-dev/chainloop/pull/1033